Data-Structures/Graph/Graph.js

```diff
@@ -67,6 +71,40 @@ class Graph {
       this.dfs(neighbour, visited, output)
     }
   }
+
+  _topologicalSort (v, visited, stack) {
+    // Mark the current node as visited.
+    visited[v] = true
+
+    // Recur for all the vertices adjacent to thisvertex
+    for (const i of this.adjacencyMap[v]) {
```

Comment:
I'd name this `v` (or `w` to not shadow the `v` in scope) since this is a vertex, not an index. Perhaps better would be `neighbor`.